overrides/android-override.json

```diff
@@ -1313,5 +1313,13 @@
                 }
             }
         ]
+    },
+    "changeOmnibarPosition": {
```

Comment:
@0nko This block must be inside the `features` block of the override file. Currently it is out side of everything so the feature still appears disabled in the android config.

Reply:
Oops, thank you! Fixed.